### PR TITLE
First set of script fixes

### PR DESCRIPTION
**Pull request by kbo001**
## Summary
Creating this PR for the follow fixes: test_cleanup_items_in_users_menu, test_case_04_deactivate_user
, test_case_04_reactivate_user, test_case_27_powerbi_tableau_integration_case, test_case_28_powerbi_tableau_integration_form

## Description
Made the below changes

- HQSmokeTests/testCases/test_02_users.py - updated test_cleanup_items_in_users_menu    
- HQSmokeTests/testPages/base/base_page.py - added new utility is_present() and wait_to_get_value()
- HQSmokeTests/testPages/data/export_data_page.py  - updated methods related to power_bi_tableau_integration_form and power_bi_tableau_integration_case
- HQSmokeTests/testPages/users/group_page.py  - added method to delete all test groups created via automation scripts
- HQSmokeTests/testPages/users/mobile_workers_page.py - updated methods related to deactivate and reactivate user

## QA Checklist

- [X] Label(s) and Assignee added
- [X] PR sent for review
- [ ] Documentation (if applicable) added/updated

### CODE REVIEW

**Review by kiran-tripathy — APPROVED** [review]

File: HQSmokeTests/testPages/users/group_page.py [hunk 1]
```diff
@@ -82,3 +82,19 @@ def cleanup_group(self):
         self.wait_to_click(self.confirm_delete)
         assert self.is_visible_and_displayed(self.delete_success_message), "Group deletion not successful"
         print("Clean up added group")
+
+    def delete_test_groups(self):
+        list_profile = self.driver.find_elements(By.XPATH,"//td//a[contains(text(),'group_')]")
```
Comment (kiran-tripathy) [inline HQSmokeTests/testPages/users/group_page.py:87]: would it make sense to move xpath` self.driver.find_elements(By.XPATH,"//td//a[contains(text(),'group_')]")` and store it in a variable?

Comment (kiran-tripathy) [reply HQSmokeTests/testPages/users/group_page.py:87]: Same for role

**Review by kiran-tripathy — APPROVED** [review]
Overall things look good, so approving.

File: HQSmokeTests/testPages/users/group_page.py (hunk 1 above)
Comment (kbo001) [reply HQSmokeTests/testPages/users/group_page.py:87]: I tried to make a variable for this one but we are using find_elements instead find_element so it kind of failed. That's why I wrote it this way.

Comment (kiran-tripathy) [reply HQSmokeTests/testPages/users/group_page.py:87]: Probably because brackets aren't required there?

Comment (kbo001) [reply HQSmokeTests/testPages/users/group_page.py:87]: which brackets? the ones in the variable declaration?

Comment (kbo001) [reply HQSmokeTests/testPages/users/group_page.py:87]: I tried both with brackets and without brackets and both times I got 
nobrackets

**Pull request merged**